### ISSUE TITLE
Pre trained weights dataset and input size

### ISSUE THREAD

**Issue by lpkoh** [Deci-AI/super-gradients#1159]
### 💡 Your Question

Hi,

I read it mentioned somewhere that Yolo-NAS's "multi-phase training process involves pre-training on Object365, COCO Pseudo-Labeled data, Knowledge Distillation (KD), and Distribution Focal Loss (DFL)". Can I check if the pretrained weights in the model zoo here: https://github.com/Deci-AI/super-gradients/blob/master/documentation/source/model_zoo.md are prerained on ALL the above?

Also, can I clarify if I can use these pretrained weights (like YOLO-NAS L) as a backbone for a model of a different input size? Like 1024 x 576. And is there a specification for what input sizes are allowed? (e.g. multiple of 32)

Lastly, can I clarify if these pretrained weights are already trained with quantization awareness, and if there is much value in adding quantization awareness in fine tuning to my tasks?

**Comment by NatanBagrov** [comment]
* The model was pretrained using the setting you mentioned, yes
* The model supports different input sizes. I do believe that it should be a multiple of 32, as the maximum downscale is 32
* The quantization awareness comes in the form of architectural structure (quantization friendly layers), so performing QAT on your dataset should help

Let me know if there are further questions

**Comment by lpkoh** [comment]
> * The model was pretrained using the setting you mentioned, yes
> * The model supports different input sizes. I do believe that it should be a multiple of 32, as the maximum downscale is 32
> * The quantization awareness comes in the form of architectural structure (quantization friendly layers), so performing QAT on your dataset should help
> 
> Let me know if there are further questions

Was it trained with those datasets though? Here (https://github.com/Deci-AI/super-gradients/issues/1143) it states that the Object365 weights were not released. So the pretrained weights are just purely trained on COCO?
